### PR TITLE
Used Makefile to get texts from Project Gutenberg and process their data

### PR DESCRIPTION
**Pull request by tslever**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# public
+output_contents_of_Makefile:
+	@cat Makefile
+
+# private
+get_text:
+	@wget https://www.gutenberg.org/cache/epub/$(text_ID)/pg$(text_ID).txt
```
Comment (EfrainOlivaresUVA) [inline Makefile:7]: ah cool, I see you are using the job to pass a variable.  Nice

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	#	$(MAKE) get_title text_ID=$$text_ID; \
+	#	$(MAKE) rename_text text_ID=$$text_ID; \
+	#	echo "Got $$(tail -n 1 Temporary_File.txt)"; \
+	#	rm Temporary_File.txt; \
+	#done
+	@bash get_the_books.sh
+
+# private
+get_title:
+	@sed -n '/^Title: /{s/^Title: //;s/\r.*//;p;q;}' pg$(text_ID).txt > Temporary_File.txt
```
Comment (EfrainOlivaresUVA) [inline Makefile:22]: incorporating sed to get the titles... 👍

File: get_the_books.sh [hunk 3]
```diff
@@ -0,0 +1,7 @@
+for text_ID in $(cat Text_IDs.csv); do
```
Comment (EfrainOlivaresUVA) [inline get_the_books.sh:1]: super combo here with the file with the book ids and a script to get them.  It's totally scalable and efficient.  I've often done this on short scripts and once in a while I next the make calls.  In this case it's totally clear, so just as an fyi, if you get another level of calling make, it's probably worth a rewrite to use direct bash.  Awesome work!

Comment (EfrainOlivaresUVA) [inline get_the_books.sh:1]: super combo here with the file with the book ids and a script to get them.  It's totally scalable and efficient.  I've often done this on short scripts and once in a while I next the make calls.  In this case it's totally clear, so just as an fyi, if you get another level of calling make, it's probably worth a rewrite to use direct bash.  Awesome work!

**Review by EfrainOlivaresUVA — APPROVED** [review]

File: Text_IDs.csv
Comment (MiaHanzhangYuan) [inline Text_IDs.csv:1]: Screenshot 2024-06-07 165329
Nice work, but you need to gather 10 books here:)

**Review by MiaHanzhangYuan — APPROVED** [review]

File: Text_IDs.csv
Comment (tslever) [reply Text_IDs.csv:1]: Please see below commit.

**Review by MiaHanzhangYuan — APPROVED** [review]

**Pull request merged**